app/core/initialize.coffee

```diff
@@ -57,6 +57,8 @@ init = ->
   setUpBackboneMediator(app)
   app.initialize()
   loadOfflineFonts() unless app.isProduction()
+  # We always want to load this font.
+  $('head').prepend '<link rel="stylesheet" type="text/css" href="/fonts/vt323.css">'
```

Comment:
Is there a reason to load this here from JS rather than putting into our base static layout file?

Reply:
I had initially reference the `loadOfflineFonts` method. However a better option is to add it to the head of only the two Vue pages that need it. Will do that and then trigger a deploy to staging.

Reply:
Punting this to next PR. Discussed at standup.